example_configs/nextcloud.md

```diff
@@ -5,8 +5,23 @@
 This example is using following users & groups in lldap :
 
 * A technical user (ex: `ro_admin`), member of `lldap_strict_readonly` or `lldap_password_manager`
-* Several accounts, members of `users` group will be authorized to log in Nextcloud (eg neither `admin` nor `ro_admin`)
-* Some "application" groups, let's say `friends` and `family`: users in Nextcloud will be able to share files and view people in dynamic lists only to members of their own group(s)
+* A group called `users`. Note: This is _not_ the same as the `Users` designation in the LLDAP Web UI. This must be a whole new group that you've created and called `users`. 
+* Members of `users` group will be authorized to log in Nextcloud (eg neither `admin` nor `ro_admin`, who are part of the `lldap_strict_readonly` or `lldap_password_manager` group.)
+* Some "application" groups, let's say `friends` and `family`: users in Nextcloud will be able to share files and view people in dynamic lists only to members of their own group(s).  
```

Comment:
I think we should clarify that users in `family` or `friends` should _also_ be in `nextcloud_users`